### PR TITLE
zh-trans: update node-conformance.md and ovs-networking.md

### PR DESCRIPTION
**Pull request by SataQiu**
/cc @xichengliudui
/assign @xichengliudui

### CODE REVIEW

File: content/zh/docs/admin/ovs-networking.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ approvers:
...
+Kubernetes 中 Vagrant 的设置如下：
 
-docker网桥被brctl生成的Linux网桥(kbr0)所代替，kbr0是具有256个地址空间的子网。总的来说，node会得到10.244.x.0/24的子网，docker上配置使用的网桥会代替默认docker0的网桥。
+docker 网桥被 brctl 生成的 Linux 网桥（kbr0） 所代替，kbr0 是具有 256 个地址空间的子网。总的来说，node 会得到 10.244.x.0/24 的子网，docker 上配置使用的网桥会代替默认 docker0 的网桥。
 
-另外，OVS网桥创建(obr0)，并将其作为端口添加到kbr0的网桥中。所有OVS网桥通过GRE隧道连接所有的nodes。因此，每个node都有一个到其他nodes的出站GRE隧道。这个隧道没有必要是一个完整的网状物，但是越像网状结构越好。在网桥上开启STP(生成树)模式以防止环路的发生。
+另外，OVS 网桥创建（obr0），并将其作为端口添加到 kbr0 的网桥中。所有 OVS 网桥通过 GRE 隧道连接所有的 nodes。因此，每个 node 都有一个到其他 nodes 的出站 GRE 隧道。这个隧道没有必要是一个完整的网状物，但是越像网状结构越好。在网桥上开启 STP （生成树）模式以防止环路的发生。
 
-路由规则允许任何10.244.0.0/16通过与隧道相连的OVS网桥到达目标。
+路由规则允许任何 10.244.0.0/16 通过与隧道相连的 OVS 网桥到达目标。
```
Comment (xichengliudui) [inline content/zh/docs/admin/ovs-networking.md]: "管道"？

Comment (SataQiu) [reply content/zh/docs/admin/ovs-networking.md]: 应该就是隧道，网络词汇
/hold cancel

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @SataQiu, there are more than 1 files in your PR, will not be reported to the task issues.

**Pull request merged**